lib/config.js

```diff
@@ -23,12 +26,18 @@ module.exports.getConfig = async function getConfig({
   context,
   getConfig: fetchConfig
 }) {
-  const defaults = {
-    ...DEFAULT_CONFIG,
-    branches: context.payload.repository.default_branch
-  }
+  const repoConfig = await fetchConfig(
+    context,
+    CONFIG_NAME,
+    DEFAULT_CONFIG
+  ).catch(error => {
+    throw error
+  })
```

Comment:
I'm wondering, is it necessary to add this? In my mind a rejected promise will throw when using `await`, right? I could very well be wrong though.

Reply:
Not necessary

Reply:
actually not unnecessary without this tests fail :)

Reply:
NVM... 😖